schema/schema_files.py

```diff
@@ -119,11 +125,11 @@
     "list_fmt": {
         "test_data": {
             "schema_file": "list_fmt/test_schema.json",
-            'prod_file': 'list_fmt.json'
+            'prod_file': 'list_fmt_test.json'
         },
         "verify_data": {
             "schema_file": "list_fmt/verify_schema.json",
-            'prod_file': 'list_fmt.json'
+            'prod_file': 'list_fmt_Verify.json'
```

Comment:
why is the "V" capitalized? is this file purely an output file or is it an input file? because if it is an input file anywhere, then a typo or misconfiguration _should_ be triggering visible problems elsewhere.

Reply:
Hmmm. I believe that the verify files are not yet being checked against the schema. This is a hole in the schema validation process.